docs/ObjC.md

```diff
@@ -106,4 +134,49 @@ Objective-C surfaced API:
 @end
 ```
 
-Here we can see that `initWithId:` has been marked as unavailable.
\ No newline at end of file
+Here we can see that `initWithId:` has been marked as unavailable.
+
+### Operator
+
+ObjC does not support operator overloading as C# does, so operators are converted to class selectors:
+
+```
+	public static AllOperators operator + (AllOperators c1, AllOperators c2)
+	{
+		return new AllOperators (c1.Value + c2.Value);
+	}
+```
+
+to
+
+```
++ (instancetype)addition:(Overloads_AllOperators *)anObjectC1 c2:(Overloads_AllOperators *)anObjectC2;
+```
+
+However, some .NET languages do not support operator overloading so it is common to also include a ["friendly"](https://msdn.microsoft.com/en-us/library/ms229032(v=vs.110).aspx) named method in addition.
+
+If both the operator version and the "friendly" version are found, only the friendly version will be generated.
+
+```
+	public static AllOperatorsWithFriendly operator + (AllOperatorsWithFriendly c1, AllOperatorsWithFriendly c2)
+	{
+		return new AllOperatorsWithFriendly (c1.Value + c2.Value);
+	}
+
+	public static AllOperatorsWithFriendly Add (AllOperatorsWithFriendly c1, AllOperatorsWithFriendly c2)
+	{
+		return new AllOperatorsWithFriendly (c1.Value + c2.Value);
+	}
+```
+
+becomes:
+
+```
++ (instancetype)addC1:(Overloads_AllOperatorsWithFriendly *)anObjectC1 c2:(Overloads_AllOperatorsWithFriendly *)anObjectC2;
```

Comment:
So out of curiosity, why do we use the "friendly" names if friendly versions are found, but not default to the friendly terms for the objc overload of an operator?  E.g. here we have the selector ```addC1:c2```, while earlier with just the + overload, we give the selector ```addition:c2:```  Why don't we use the "friendly" names (Add, Subtract, Multiply, Divide) for the class selector if they only override the operator?